crates/oxc_transformer/src/plugins/styled_components.rs

```diff
@@ -701,6 +706,15 @@ impl<'a> StyledComponents<'a, '_> {
                 file_stem
             };
 
+            ctx.ast.allocator.alloc_str(block_name)
```

Comment:
There's an unnecessary allocation in `to_string` in the block above:

```diff
- let block_name = if self.options.meaningless_file_names.contains(&file_stem.to_string())
+ let block_name = if self.options.meaningless_file_names.iter().any(|file_name| file_stem == file_name)
```

Reply:
Yes, `meaningless_file_names` has a few items always, so this is fine to use `iter.any`

Reply:
I just realized that 😅